getting-started-gradle/src/test/java/quickstart/NativeGreetingResourceIT.java

```diff
@@ -0,0 +1,9 @@
+package quickstart;
+
+import io.quarkus.test.junit.SubstrateTest;
+
+@SubstrateTest
+public class NativeGreetingResourceIT extends GreetingResourceTest {
```

Comment:
Did you try to build the project with native? One of the reason we wanted a separate project is that, AFAIK, the IT tests should be in a separate directory and run separately from the unit tests.

Reply:
One of the reason it might not fail is because we haven't merged `https://github.com/quarkusio/quarkus-quickstarts/pull/139/files` yet.

See this comment here: https://github.com/quarkusio/quarkus/issues/1930#issuecomment-481179812 , especially about how to separate the tests.

We might also need to upgrade the Gradle project template in Quarkus itself.